micromasters/settings.py

```diff
@@ -83,8 +84,26 @@ def get_var(name, default):
     'rest_framework',
     'server_status',
     'social.apps.django_app.default',
+
+    # WAGTAIL
+    'wagtail.wagtailforms',
+    'wagtail.wagtailredirects',
+    'wagtail.wagtailembeds',
+    'wagtail.wagtailsites',
+    'wagtail.wagtailusers',
+    'wagtail.wagtailsnippets',
+    'wagtail.wagtaildocs',
+    'wagtail.wagtailimages',
+    'wagtail.wagtailsearch',
+    'wagtail.wagtailadmin',
+    'wagtail.wagtailcore',
+
+    'modelcluster',
+    'taggit',
```

Comment:
Nit : are these still for  wagtail? If so why the space? Otherwise could you put a comment for what they are?